test/unit/test_server.py

```diff
@@ -34,8 +34,7 @@
 
 
 def test_api_compatible():
-    '''
-    Test whether `swift_scality_backend.server.app_factory`'s result is
+    '''Test whether `swift_scality_backend.server.app_factory`'s result is
```

Comment:
Hmpf, not very fond of this 'rule' then... If something fits on a single line, having it at the same line as the quotes, OK. But this looks a bit distracting.